linera-core/src/unit_tests/client_test_utils.rs

```diff
@@ -260,6 +262,7 @@ where
     }
 }
 
+#[derive(Clone)]
```

Comment:
@afck Do we need this one?

Reply:
Yes, because the chain client builder has to clone the validator node provider for every chain client it builds.